api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -147,15 +147,14 @@ public static Meter getMeter(String instrumentationName, String instrumentationV
   }
 
   /**
-   * Returns a singleton {@link CorrelationContextManager}.
+   * Returns a singleton {@link BaggageManager}.
    *
-   * @return registered manager or default via {@link
-   *     DefaultCorrelationContextManager#getInstance()}.
+   * @return registered manager or default via {@link DefaultBaggageManager#getInstance()}.
    * @throws IllegalStateException if a specified manager (via system properties) could not be
    *     found.
    * @since 0.1.0
```

Comment:
Hmm - looks like we only removed these in SDK in #344 but we should remove them everywhere